Source/carthage/Update.swift

```diff
@@ -73,22 +72,13 @@ public struct UpdateCommand: CommandProtocol {
 				<*> mode <| Option(key: "log-path", defaultValue: nil, usage: "path to the xcode build output. A temporary file is used by default")
 				<*> mode <| Option(key: "new-resolver", defaultValue: false, usage: "use the new resolver codeline when calculating dependencies. Default is false")
 				<*> BuildOptions.evaluate(mode, addendum: "\n(ignored if --no-build option is present)")
-				<*> CheckoutCommand.Options.evaluate(mode, useBinariesAddendum: binariesAddendum, dependenciesUsage: dependenciesUsage)
+				<*> CheckoutCommand.Options.evaluate(mode, dependenciesUsage: dependenciesUsage)
 		}
 
 		/// Attempts to load the project referenced by the options, and configure it
 		/// accordingly.
 		public func loadProject() -> SignalProducer<Project, CarthageError> {
 			return checkoutOptions.loadProject()
-				.on(value: { project in
-					// Never check out binaries if 
-					// 1. we're skipping the build step, or
-					// 2. `--toolchain` option is given
-					// because that means users may need the repository checkout.
-					if !self.buildAfterUpdate || self.buildOptions.toolchain != nil {
-						project.useBinaries = false
-					}
-				})
```

Comment:
I think we need to keep this logic, even though `--use-binaries` has moved to `build`.

Reply:
Why do you think so? From now it always checkouts repositories though.

Reply:
Oh, I guess you're right!

Reply:
NP!